evalml/utils/gen_utils.py

```diff
@@ -270,3 +270,28 @@ def is_all_numeric(df):
         if dtype not in numeric_and_boolean_dtypes:
             return False
     return True
+
+
+def _convert_woodwork_types_wrapper(pd_data):
+    """
+    Converts a pandas data structure that may have extension or nullable dtypes to dtypes that numpy can understand and handle.
+
+    Arguments:
+        pd_data (pd.Series, pd.DataFrame, pd.ExtensionArray): Pandas data structure
+
+    Returns:
+        New pandas data structure (pd.DataFrame or pd.Series) with original data and dtypes that can be handled by numpy
+    """
+    nullable_to_numpy_mapping = {pd.Int64Dtype: 'int64',
+                                 pd.CategoricalDtype: 'category',
```

Comment:
Isn't this the same thing? categorydtype is `category`?

Reply:
Yup, they're equivalent! I added it to handle the `pd.Categorical()` case (where it'll create a Series and use the mapped type as the type), but I can update this to only change the dtype if necessary.